Features/DataMIDI.py

```diff
@@ -0,0 +1,27 @@
+from mido import MidiFile, tempo2bpm
+import FeaturesMIDI
+
+
+class File:
+    def __init__(self, file_name):
+        self.mid = MidiFile(file_name)
+        self.features = FeaturesMIDI.FeaturesMIDI()
+        self.parseData()
+
+    def parseData(self):
+        for i, track in enumerate(self.mid.tracks):
+            # print('Track {}: {}'.format(i, track.name))
+            for msg in track:
+                if msg.type == 'set_tempo':
+                    # fill tempo
+                    self.features.tempos.append(round(tempo2bpm(msg.tempo)))
+                    # fill tempo with repetition
+                    find = False
+                    for j in range(len(self.features.temposWithRepetition)):
+                        if self.features.temposWithRepetition[j].get('tempo') == round(tempo2bpm(msg.tempo)):
+                            self.features.temposWithRepetition[j]['repetition'] = self.features.temposWithRepetition[j][
+                                                                                      'repetition'] + 1
```

Comment:
It's cool but then you extract a note counter and if we end up with longer music we'll get a lot more notes. So the number of notes will depend a lot on the size of the music. I think you have to divide each repetition by the total number of notes to get a percentage. And rename the counter as an average. WDYT?

Reply:
Careful, this is not about notes, it's about `tempo` MIDI messages. The point is still valid but we should take into account the time in this computation.
If we want to get `percentage of time where the tempo was n BPM`, the time must be taken into account.